src/haxeLanguageServer/features/haxe/completion/CompletionFeature.hx

```diff
@@ -146,10 +146,10 @@ class CompletionFeature {
 		final data:Null<CompletionItemData> = item.data;
 		if (!context.haxeServer.supports(DisplayMethods.CompletionItemResolve)
 			|| previousCompletionData == null
-			|| data!.origin == Custom) {
+			|| data?.origin == Custom) {
 			return resolve(item);
 		}
-		final index = data!.index.sure();
+		final index = (data?.index).sure();
```

Comment:
Checking what's happening there :thinking:

Reply:
I think this is `if data is null return null, else call data.index.sure()`, so index is nullable here without brackets. Idk what can be changed here in call logic, this is why other null-safety operators exist maybe.

Reply:
It can be also written as `data.sure().index` i guess

Reply:
Guess it's fine; not sure why safety macro was ok with previous version :shrug: